detectable/src/main/java/com/synopsys/integration/detectable/detectables/gradle/inspection/parse/GradleReportLineParser.java

```diff
@@ -27,9 +25,10 @@ public class GradleReportLineParser {
     private static final String REQUIRE = "require";
     private static final String PREFER = "prefer";
     private static final String REJECT = "reject";
-    private static final Map<String, String> gradleRichVersions = new HashMap<>();
+    private static final Map<String, Map<String, String>> gradleRichVersions = new HashMap<>();
```

Comment:
We might benefit from naming this variable in a way that reveals what the different indices and values in the map mean. Or perhaps, we can add a comment describing this.